packages/mux-player/src/utils.ts

```diff
@@ -21,3 +21,15 @@ export function toNumberOrUndefined(val: any) {
   const num = +val;
   return !Number.isNaN(num) ? num : undefined;
 }
+
+export function toQuery(obj: Record<string, any>) {
+  const params = toParams(obj).toString();
+  return params ? "?" + params : "";
+}
+
+export function toParams(obj: Record<string, any>) {
+  for (let key in obj) {
+    if (obj[key] == null) delete obj[key];
```

Comment:
blocker: this will mutate the object passed in. If we do want to handle automatically pruning null/undefined (not sure we do, but can see an argument for it), we should probably do it in a non-mutative way, e.g.:
```js
export function toParams(obj: Record<string, any>) {
  const objSansNils = Object.entries(obj).reduce<Record<string, any>>(
    (objSansNils, [k, v]) => {
      if (v != null) {
        objSansNils[k] = v;
      }
      return objSansNils;
    },
    {}
  );
  return new URLSearchParams(objSansNils);
}
```

Reply:
fixed in [`1c0cc70` (#107)](https://github.com/muxinc/elements/pull/107/commits/1c0cc703ca9756f8257c13386ab61c3846d1189c)